src/Nethermind/Nethermind.Consensus/Processing/BlockchainProcessor.cs

```diff
@@ -72,7 +72,7 @@ public sealed class BlockchainProcessor : IBlockchainProcessor, IBlockProcessing
 
     private int _currentRecoveryQueueSize;
     private bool _isProcessingBlock;
-    private const int MaxBlocksDuringFastSyncTransition = 8192;
+    private const int MaxBranchSize = 8192;
```

Comment:
Should this be same as `Reorganization.PersistanceInterval` or maybe better `PruningConfig.PersistanceInterval`?

Reply:
We are even using `Reorganization.PersistenceInterval)` in `PrepareProcessingBranch` here